packager/packager.cc

```diff
@@ -182,6 +182,11 @@ class PackageDB final {
             curPrefixPos = path.find_last_of('/', curPrefixPos - 1);
         }
 
+        const auto &root = packageInfoByPathPrefix.find("");
```

Comment:
This returns an iterator, I'm not sure it needs the `&`.
```suggestion
        const auto root = packageInfoByPathPrefix.find("");
```